en-US/win10/samples/ZigBeeAdapterTutorial.md

```diff
@@ -0,0 +1,203 @@
+
+![AJXPhilipsHue1]({{site.baseurl}}/images/ZigBee/AJXPhilipsHue1.png)
+
+![AJXPhilipsHue2]({{site.baseurl}}/images/ZigBee/AJXPhilipsHue2.png)
+
+![AJXPhilipsHue3]({{site.baseurl}}/images/ZigBee/AJXPhilipsHue3.png)
+
+![AJXPhilipsHue4]({{site.baseurl}}/images/ZigBee/AJXPhilipsHue4.png)
+
+![AJXPhilipsHue5]({{site.baseurl}}/images/ZigBee/AJXPhilipsHue5.png)
+
+### Class overview
+
+![ClassMap]({{site.baseurl}}/images/ZigBee/ClassMap.png)
+
+The __Adapter__ class is the main class of ZigBee adapter. This class derives from __IAdapter__ (BridgeRT interface) and contains a collection of __ZigBeeDevice__ instances and an instance of the __XBeeModule__ class.
+
+The __XBeeModule__ class handles the interactions with XBee module, this class has method to build and parse XBee frames. See XBee ZigBee module documentation (http://www.digi.com) for more detail about its API and its frame layout. XBeeModule class uses the __SerialController__ class that handles communication over a serial port. 
+
+The __ZigBeeDevice__ class represent a ZigBee device. This class has no interface with BridgeRT since only EndPoints of a ZigBee device are expose to AllJoyn. __ZigBeeDevice__ class contains a collection of __ZigBeeEndPoint__ instances.
+
+The __ZigBeeEndPoint__ class represent an EndPoint of a ZigBee device. This class derives from __IAdapterDevice__ (BridgeRT interface) and contains an instance of the __BasicCluster__ and a collection __ZclCluster__ instance. To be more accurate it contains a collection of cluster classes, e.g.: __OnOffCluster__, which derive from the abstract __ZclCluster__ class.
+
+The cluster classes, e.g.: __BasicCluster__, __OnOffCluster__, represent a ZCL cluster. Each cluster class is a “specific” implementation of the __ZclCluster__ abstract class.
+ 
+The __ZclCluster__ class is an abstract class that derives from __IAdapterProperty__ (BridgeRT interface) and contains a collection of __ZclAttribute__ instances and a collection of __ZclCommand__ instances. A specific implementation of ZclCluster simply consists in defining the list of ZclAttribute and ZclCommand that should be supported for that specific cluster.
+
+The __ZclAttribute__ class represent an attribute as defined in ZCL standard. This class derives from __IAdapterAttribute__ (BridgeRT interface) and from __ZigBeeCommand__ class. This class contains an instance of __IAdapterValue__ class which is a BridgeRT interface. For information, reading or writing an attribute consist in sending a specific ZCL command.
+
+The __ZclCommand__ class represent a command as defined in ZCL standard. This class derives from __IAdapterMethod__ (BridgeRT interface) and from __ZigBeeCommand__ class. This class contains list of input and output parameters. These parameters are actually __IAdapterValue__ class which is a BridgeRT interface.
```

Comment:
Suggested grammar fixes:
The __ZclCommand__ class represents a command as defined in ZCL standard. This class derives from __IAdapterMethod__ (BridgeRT interface) and the __ZigBeeCommand__ class